mappings/net/minecraft/world/gen/feature/DefaultBiomeFeatures.mapping

```diff
@@ -321,15 +321,15 @@ CLASS net/minecraft/class_3864 net/minecraft/world/gen/feature/DefaultBiomeFeatu
 		ARG 0 biome
 	METHOD method_24384 addWarpedForestVegetation (Lnet/minecraft/class_1959;)V
 		ARG 0 biome
-	METHOD method_28437 (Lnet/minecraft/class_1959;)V
+	METHOD method_28437 addMesaStructures (Lnet/minecraft/class_1959;)V
 		ARG 0 biome
 	METHOD method_28438 addNetherOres (Lnet/minecraft/class_1959;II)V
 		ARG 0 biome
 		ARG 1 goldCount
 		ARG 2 quartzCount
 	METHOD method_28439 addAncientDebris (Lnet/minecraft/class_1959;)V
 		ARG 0 biome
-	METHOD method_28440 (Lnet/minecraft/class_1959;)V
+	METHOD method_28440 addDefaultStructures (Lnet/minecraft/class_1959;)V
 		ARG 0 biome
-	METHOD method_28441 (Lnet/minecraft/class_1959;)V
+	METHOD method_28441 addOceanStructures (Lnet/minecraft/class_1959;)V
```

Comment:
These are underground structures only, not all structures. Also, mesa is now badlands, even if the mineshaft type is still called "mesa"

Reply:
I changed the default and badlands ones to `-UndergroundStructures`, and fixed the badlands biome name. I left the ocean one as it was as the sunken ships aren't underground.